### DISCUSSION TITLE
Handle a-z keys in Select dropdown the same way as in native select element

### DISCUSSION THREAD

**Discussion by frec-hansen** [mantinedev/mantine#2077] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

With native select, once the dropdown is shown, a user can navigate quickly using the keyboard `a-z` to jump to value. But this is not the case with select.

### In which browser did the problem occur

Chrome latest

### Do you know how to fix the issue

No

**Comment by rtivital** [comment]
Most likely this feature will never be added because select can be searchable. If you need this feature, you should use NativeSelect component.
